source/functions/initializeWecSim.m

```diff
@@ -382,6 +382,17 @@
 %    eval(['sv_b' num2str(ii) '_rigidBody = Simulink.Variant(''nhbody_' num2str(ii) '==0'');'])
 end; clear ii
 
+% Visualization Blocks
+
+if ~isempty(waves.waveAmpTimeViz)
```

Comment:
@salhus Based on your last comment on this PR you were removing this variable; however, when I pull your latest files this is still present. Did you push you latest changes?